.markdownlint.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# Use `#` for headers
+MD003:
+  style: atx
+
+# Set maximum line length
+MD013:
+  line_length: 150
+
+# Use `---` for horizontal rule
+MD035:
+  style: ---
+
+# Use ``` for code blocks
+MD046:
+  style: fenced
+MD048:
+  style: backtick
```

Comment:
This might not work because our auto generated bindings.md is using `<code>` to get around key that ends with <code>\`</code>

Reply:
yes! That file is ignored now :)